app/Http/routes.php

```diff
@@ -42,6 +42,7 @@
     Route::get('messages/{message}/send', 'MessagesController@sendMessage')->name('messages.send');
 
     // Users
+    Route::get('users/contestants', 'UsersController@contestantIndex')->name('users.contestants');
```

Comment:
Ya. There's nothing else that matches it, so if the second item in a url param to `/users/?` doesn't match `contestant` then it moves along to next route which is the resource.